src/main/java/com/pinterest/secor/message/Message.java

```diff
@@ -20,55 +20,75 @@
 import java.io.OutputStream;
 import java.lang.String;
 
+
 /**
  * Message represents a raw Kafka log message.
  *
  * @author Pawel Garbacki (pawel@pinterest.com)
  */
 public class Message {
+
```

Comment:
Sorry our cooperate standard has some double blank lines.  What is the best way for me to re-submit this.  I am pretty new to git-hub.

Leo